test/functional/apps/dashboard/group6/create_and_add_embeddables.ts

```diff
@@ -1,169 +0,0 @@
-/*
```

Comment:
Was this test file duplicated / not used in `group6`? Or was this accidentally deleted?

Reply:
Perfect, that's what I figured. Thanks for the clarification!